### DISCUSSION TITLE
Chyrp-lite : how to hide post from  main page?

### DISCUSSION THREAD

**Discussion by supersophie** [xenocrat/chyrp-lite#214] [Q&A] [answered]
Hi everybody,

I want to create a post or have a photo on Chyrp, but not have it visible on the main page/blog that everybody can see.

Then, I want to put the URL on another website, that anybody can access it from.

How can I do this?

**Comment by xenocrat** [comment, marked as answer]
You can't do this with a post, but you can do it with a page: set the status to "Public" and it will be accessible but not listed.

**Reply by supersophie** [reply]
Thank-you!
